device-types/Cisco/C9200L-24PXG-2Y.yaml

```diff
@@ -0,0 +1,128 @@
+---
+manufacturer: Cisco
+model: C9200L-24PXG-2Y
+part_number: C9200L-24PXG-2Y
+slug: cisco-c9200l-24pxg-2y
+u_height: 1
+is_full_depth: false
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
Hello and thank you for reviewing! I've added the requested data points and module bays, as well as the supported (AC) PSU.